src/IO/WriteHelpers.h

```diff
@@ -581,6 +581,59 @@ void writeCSVString(const StringRef & s, WriteBuffer & buf)
     writeCSVString<quote>(s.data, s.data + s.size, buf);
 }
 
+inline void writeHTMLString(const char * begin, const char * end, WriteBuffer & buf)
```

Comment:
The name is kind-of misleading.
The function is equally applicable to XML and HTML,
but unlike the `writeXMLString`, it also allows to place data into attributes.

We need to invent some better name for it...

Reply:
I am Sorry. Would it be better to use writeXMLOrHTMLString?

Reply:
I am not very clear about the use case.

Reply:
Let's rename the old functon to `writeXMLStringForTextElement` and new function to `writeXMLStringForTextElementOrAttributeValue`.

Then we need to implement SQL function to use it.

Reply:
thank you for your advice.